core/foundation/v7/inc/ROOT/RError.hxx

```diff
@@ -61,7 +61,15 @@
 //           return R__FAIL("user-facing error messge");
 //        return RResult<void>::Success();
 //     }
-
+//
```

Comment:
It'd be nice to have this captured as documentation, I suppose? Check the [Doxygen doc](https://root.cern/doc/master/classROOT_1_1Experimental_1_1RError.html) - it's not there because it's not commented with `///`, and not in front of the class.